ScoutSuite/__init__.py

```diff
@@ -1,15 +1,9 @@
 # -*- coding: utf-8 -*-
 
 __author__ = 'NCC Group'
-__version__ = '5.0.0-beta'
+__version__ = '5.0.0'
```

Comment:
Not sure this PR promotes the version to a stable version 😆 

```suggestion
__version__ = '5.0.0-beta'
```